_posts/news/2017-03-21-announcing-scalajs-0.6.15.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.15
+category: news
+tags: [releases]
+permalink: /news/2017/03/21/announcing-scalajs-0.6.15/
+---
+
+
+We are excited to announce the release of Scala.js 0.6.15!
+
+This release was focused on preparations for Scala.js 1.0.0.
+In particular, it should contain the last deprecations for things that are being replaced by better alternatives.
+If your code and build compile without warnings with 0.6.15, it should continue to work unchanged in 1.0.0.
```

Comment:
The `jsDependencies` will move to a separate sbt plugin where we can afford to keep crappy things forever ^^